db/migrate/20241105151352_create_project_life_cycle_steps.rb

```diff
@@ -0,0 +1,15 @@
+class CreateProjectLifeCycleSteps < ActiveRecord::Migration[7.1]
+  def change
+    create_table :project_life_cycle_steps do |t|
+      t.string :type
+      t.date :start_date
+      t.date :end_date
+      t.boolean :active, default: false, null: false
+      t.integer :position, default: 1, null: true
```

Comment:
If this is for ordering when listing, then I think it should either follow the order of definitions or use `start_date`

Reply:
Yes, this is for the acts_as_list plugin. The default ordering follows the order of definition, however the list can be drag-and-drop reordered from the admin interface.

After running the seeders, the order is set correctly:

```ruby
[4] pry(main)> Project::LifeCycleStepDefinition.all.pluck(:id, :name, :position)
  Project::LifeCycleStepDefinition Pluck (0.8ms)  SELECT "project_life_cycle_step_definitions"."id", "project_life_cycle_step_definitions"."name", "project_life_cycle_step_definitions"."position" FROM "project_life_cycle_step_definitions"
=> [[1, "Initiating", 1], [2, "Ready for Planning", 2], [3, "Planning", 3], [4, "Ready for Executing", 4], [5, "Executing", 5], [6, "Ready for Closing", 6], [7, "Closing", 7]]
```

Reply:
But what would be the point of reordering? Names are coming from definitions, and it make sense to be able to reorder them, but why changing order of date assignments for specific project?

Reply:
Sorry I misunderstood your initial question. Yes, does not makes sense to have the list in the `Project::LifeCycleStep`, it is sufficient for the `Project::LifeCycleStepDefinition`.